### PR TITLE
 feat(export): Add export of transcripts (if available) to BMM 

### PR DESCRIPTION
**Pull request by KillerX**
Note: I did verify with BMM that adding a field to the object will *not* break import, the extra field will just be ignored!

Possible future improvements:

* Try a "real" (not auto transcribed) subtitle file first. How will this work with word accurate timestamps?
* Force transcription and wait for it if not available
* Extra languages

### CODE REVIEW

File: activities/transcribe.go [hunk 1]
```diff
@@ -50,3 +53,113 @@ func Transcribe(
...
+				word.End += startAt
+			}
+
+			mergedTranscription.Segments = append(mergedTranscription.Segments, segment)
+			mergedTranscription.Text += segment.Text + " "
+		}
+
+		startAt += mi.End - mi.Start
+	}
+
```
Comment (fredrikved) [inline activities/transcribe.go:149]: Should we move this to services.transcribe?

File: workflows/export/merge_export_data.go [hunk 2]
```diff
@@ -6,39 +6,49 @@ import (
...
 type MergeExportDataResult struct {
-	Duration      float64
-	VideoFile     *paths.Path
-	AudioFiles    map[string]paths.Path
-	SubtitleFiles map[string]paths.Path
+	Duration       float64
+	VideoFile      *paths.Path
+	AudioFiles     map[string]paths.Path
+	SubtitleFiles  map[string]paths.Path
+	JSONTranscript *paths.Path
```
Comment (fredrikved) [inline workflows/export/merge_export_data.go:19]: Maybe this should be a map as well to support other languages in the future.

**Review by fredrikved — APPROVED** [review]

File: activities/transcribe.go (hunk 1 above)
Comment (KillerX) [reply activities/transcribe.go:149]: Yeah, that makes sense.

**Comment by KillerX** [comment]
I'll wait with this merge until after the WE.

File: activities/transcribe.go [hunk 3]
```diff
@@ -50,3 +51,42 @@ func Transcribe(
...
+	marshalled, err := json.Marshal(mergedTranscription)
+	if err != nil {
+		return nil, err
+	}
+
+	WriteFile(ctx, WriteFileInput{
+		Path: targetFile,
+		Data: marshalled,
+	})
```
Comment (fredrikved) [inline activities/transcribe.go:85]: This also returns an error if it fails. Might be smart to return it here if it occurs.

**Pull request merged**